source/data-channel.js

```diff
@@ -180,6 +180,40 @@ Skylink.prototype._createDataChannel = function(peerId, dataChannel, bufferThres
   }
 };
 
+/**
+ * Function that refreshes the main messaging Datachannel.
+ * @method refreshDatachannel
+ * @private
```

Comment:
Why is this `@private` given that it is a public interface for developers?

Reply:
Additionally, there doesn't seem to have the documented line for the peerId parameter. Reference `refreshConnection()` documentation (in `source/peer-connection.js`) for information.